visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.flatten.component.html

```diff
@@ -17,7 +17,7 @@ <h3 class="header-text">Flattened</h3>
 			</div>
 		</md-list-item>
 
-		<md-list-item ng-repeat="item in $ctrl._viewModel.flatten | orderBy: ['path'] track by $index" title="{{::item.path}}">
+		<md-list-item ng-repeat="item in $ctrl._viewModel.flatten | orderBy: ['path'] track by $index+item.path" title="{{::item.path}}">
```

Comment:
When using `track-by`, we need to track by something that is actually unique. AngularJS usually applies a hash to each object used in `ng-repeat`, but with `track-by` we have to deal with this by ourselves. `track by $index` is used, when you have a list with duplicates. Usually, angularjs would throw an error since duplicates result in the same hash, but with `$index` the hash is now the index. 

For some reason, if only the index is used here, we will have a blacklist in our view where each entry has the same path, even though they're different during runtime. They're just not displayed correctly. I don't know, why this is the case in this branch and not on master.